osmosis/src/setupStore.js

```diff
@@ -25,8 +25,9 @@ const setupStore = useCustomHook => {
       </StoreContext.Provider>
     );
   };
-
-  return [StoreContext, withStoreContext, storeRef];
+  storeRef.Context = StoreContext;
+  storeRef.Provider = withStoreContext;
+  return [ storeRef ];
```

Comment:
no need to have this return an array, since its just a single object at this point